deps/wazuh_testing/wazuh_testing/tools/system.py

```diff
@@ -62,16 +65,16 @@ def add_block_to_file(self, host: str, path: str, replace: str, before: str, aft
         self.get_host(host).ansible("replace", fr"path={path} regexp='{after}[\s\S]+{before}' replace='{replace}'",
                                     check=check)
 
-    def modify_file_content(self, host: str, path: str = None, content: str = ''):
+    def modify_file_content(self, host: str, path: str = None, content: Union[str, bytes] = ''):
```

Comment:
Actually the default behavior remains the same as before. It is, when `content` is not an instance of `bytes` (there isn't any call to this function where content is bytes, until now), this will be used: `content.encode()`. So I think it is not necessary in this case.